mxv/templates/mxv/index.html

```diff
@@ -46,7 +46,7 @@ <h3>Nominations close Thursday 11th June</h3>
                 <p>Momentum’s leadership is directly elected by its members every two years. From political education to supporting trade
                 union struggles and training a new generation of socialist leaders, there is so much Momentum could do. Now is your
                 chance to get involved and help shape the future of our movement.</p>
-                <a href="{% url 'ncg_election' %}" class="btn btn-outline-secondary btn-block">Read More</a>
+                <a target="_blank" href="http://vote.peoplesmomentum.com/info" class="btn btn-outline-secondary btn-block">Read More</a>
```

Comment:
will leave for now but in general this sort of thing should come from the NCG_VOTING_URL environment variable